ksmiahlikau/01.HTML/Task_2.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>  
+	<HTML>
+		<HEAD>
+			<META charset="utf-8">
+			<TITLE>Пример страницы</TITLE>
+		</HEAD>
+<!--Начало документа-->  
+		<BODY> 
+			<!--заголовок-->
+			<H1>HOW-TO: Программа на Python, Часть 1</H1>
+			<!--маркированный список--> 
+			<ul><li>Программа на Python — часть 1</li>
+			<li>Программа на Python — часть 2</li>
+			<li>Программа на Python — часть 3</li></ul>
+			<!--заголовок-->
+			<H2>Содержание</H2>
+			<H3>HOW-TO: Программа на Python, Часть 1</H3>
+			<!--маркированный список-->
+			<ul><li>Наша первая программа</li>
+			<li>Пример цикла For</li></ul>
+			<!--текст абзаца.-->		    
+			<P>Из множества существующих языков программирования Python — один из самых лёгких в изучении. Он создан в конце 80-х годов XX века, но с тех пор он сильно изменился. Хотя Python входит в состав большинства дистрибутивов Linux, на него часто не обращают внимания при выборе языка для изучения. В этой статье мы рассмотрим создание консольных приложений, а в следующей — программирование графических интерфейсов (GUI). Давайте создадим простую программу прямо сейчас.</P>		 
+			<!--заголовок-->		
+			<H2>Наша первая программа</H2>		
+			<!--текст-->		
+			<P>Наберём код в текстовом редакторе (например, в gedit). Потом мы разберёмся, что делает каждая строка и пойдём дальше.</P>
+			<P>Введите эти 4 строки:</P>
+			<P><code><!--код программы-->
+				#!/usr/bin/env python<br>
+				print 'Hello. I am a python program.'<br>
+				name = raw_input("What is your name?")<br>		
+				print "Hello there, " + name + "!"<br></code></P>			
+			<P>Это всё. Сохраните файл как hello.py в любом месте. Я предлагаю сохранить его в папке
+				python_examples в домашнем каталоге. Этот пример показывает, как просто написать программу на
+				Python. Для запуска программы нужно сделать её файл исполняемым. Для этого введите в терминале</P>
+				<!--код программы-->
+			<p><code>chmod +x hello.py</code></p>
+			<p>в папке, в которой вы сохранили программу. Теперь запустите её.</p>
+			<p><code>greg@earth:~/python_examples$ ./hello.py<br>	
+				Hello. I am a python program.<br>	
+				What is your name? Ferd Burphel<br>	
+				Hello there, Ferd Burphel!<br>	
+				greg@earth:~/python_examples$</code></p>	
+			<p>Вот и всё.</p>		
+			<H2>Пример цикла For</H2>	
+			<p>Теперь давайте рассмотрим пример программирования цикла. Запустите текстовый редактор и наберите следующую программу:</p>
+			<p><code><pre>
```

Comment:
Do not use tag p for not human text
and shift tag code into tag pre

Reply:
ok fixed